app/preprints/-components/submit/supplements/component.ts

```diff
@@ -11,24 +13,69 @@ interface SupplementsArgs {
     manager: PreprintStateMachine;
 }
 
+interface SupplementsForm{
+    node: Node;
+}
+
+const SupplementFormValidation: ValidationObject<SupplementsForm> = {
+    node: validatePresence({
+        presence: true,
+        ignoreBlank: true,
+        type: 'empty',
+    }),
+};
+
 /**
  * The Supplements Component
  */
 export default class Supplements extends Component<SupplementsArgs>{
-    @service intl!: Intl;
+    @tracked displayExistingNodeWidget = false;
+    @tracked displayCreateNodeWidget = false;
+    @tracked displayCancelButton = false;
+
+    supplementFormChangeset = buildChangeset(this.args.manager.preprint, SupplementFormValidation);
+
+    private isDisplayCancelButton(): void {
+        this.displayCancelButton = this.displayExistingNodeWidget ||
+        this.displayCreateNodeWidget;
+    }
```

Comment:
Can this method and `this.displayCancelButton` be combined into a single getter? getters should auto-update if the tracked properties they rely on are updated, and that way we don't have to call this method to update `this.displayCancelButton`